src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -168,6 +176,69 @@ private IEnumerable<CompletionItem> GetResourceTypeCompletions(SemanticModel mod
                 .ToList();
         }
 
+        private IEnumerable<CompletionItem> GetModulePathCompletions(SemanticModel model, BicepCompletionContext context)
+        {
+            if (!context.Kind.HasFlag(BicepCompletionContextKind.ModulePath)
+            || FileResolver.TryResolveModulePath(model.SyntaxTree.FileUri, ".")?.LocalPath is not string cwd
```

Comment:
[nit] To be consistent with the style used elsewhere in this repo, please indent `||` and `&&`:
```csharp
if (abc 
    && def
    || ghi)
{
    ...
}
```